bc_obps/registration/utils.py

```diff
@@ -118,3 +135,126 @@ def extract_fields_from_dict(data_dict, fields_to_extract):
         if field in data_dict:
             new_dict[field] = data_dict[field]
     return new_dict
+
+
+def raise_401_if_role_not_authorized(request, authorized_roles) -> Tuple[int, Optional[Union[dict[str, str], None]]]:
+    if not hasattr(request, 'current_user'):
+        raise HttpError(401, "Unauthorized.")
+    role_name = getattr(request.current_user.app_role, "role_name")
+    if role_name not in authorized_roles:
+        raise HttpError(401, "Unauthorized.")
+
+
+def get_an_operators_users(operator: Operator) -> List[User]:
```

Comment:
nit: could we rename this to `get_an_operators_approved_users` for extra clarity please?